doc/api/util.md

```diff
@@ -377,8 +377,8 @@ terminals.
 <!-- type=misc -->
 
 Objects may also define their own `[util.inspect.custom](depth, opts)`
-(or, equivalently `inspect(depth, opts)`) function that `util.inspect()` will
-invoke and use the result of when inspecting the object:
+(or, equivalently but deprecated, `inspect(depth, opts)`) function that
```

Comment:
not sure, but maybe "equivalent but deprecated"?

Reply:
Yes, you're correct. Even better might be `or the equivalent but deprecated` which is what I'll switch it to in a minute.